src/routes/api/bookings/queryBookings.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import Booking from '../../../models/booking.model';
+
+/**
+ * Route to query all bookings or the bookings from one booker
+ * @author Kevin Wang
+ */
+const queryBookingsRoute = async (req: Request, res: Response): Promise<void> => {
+    const bookerEmail = req.query.id;
+    if (bookerEmail) {
+        if (typeof bookerEmail === 'string') {
+            // If an booker's discord id is included in the query, retrieve the bookings of the and return the booking
+            const bookings = await Booking.find({ booker: bookerEmail }).catch((error) => {
+                res.status(404).json({ error });
```

Comment:
`404` on an arbitrary error doesn't make sense, and the return statement on the next line will only exit the promise.